source/connect.txt

```diff
@@ -11,147 +11,15 @@ Connect to MongoDB
-   connection.
-
-Delete a Favorite Connection
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-To delete a saved favorite connection:
-
-1. Load the connection into the :guilabel:`Connect` form.
-
-#. Click :guilabel:`Delete Favorite` at the bottom of the form.
-
-|compass| removes the connection from the :guilabel:`Favorites` list
-and returns to the :guilabel:`New Connection` view.
-
-.. _required-access:
-
-Required Access
----------------
-
-With authentication/authorization enforced on the connected MongoDB
-instance, the user requires appropriate privileges to access various
-features. The following table lists the privileges required to access
-the features as well as the built-in roles that can provide these
-privileges:
-
-.. note::
-   The built-in roles may provide more access than required. You can
-   also create a :ref:`user-defined-roles` to grant specific privileges.
-
-.. list-table::
-   :header-rows: 1
-   :widths: 25 30 25 20
-
-   * - View
-     - Feature/Action
-     - Privilege(s)
-     - MongoDB Built-In Role
-
-   * - :guilabel:`Home/MongoDB Instance`
-     - :doc:`/performance`
-     - | :authaction:`top`
-       | :authaction:`inprog`
-       | :authaction:`serverStatus`
-     - :authrole:`clusterMonitor`
-
-   * - :guilabel:`Database`
-     - :ref:`create-database`
-     - :authaction:`createCollection`
-     - :authrole:`readWrite`
-
-   * - :guilabel:`Database`
-     - :ref:`drop-database`
-     - :authaction:`dropDatabase`
-     - :authrole:`dbAdmin`
-
-   * - :guilabel:`Collection`
-     - :ref:`create-collection`
-     - :authaction:`createCollection`
-     - :authrole:`readWrite`
-
-   * - :guilabel:`Collection`
-     - :ref:`drop-collection`
-     - :authaction:`dropCollection`
-     - :authrole:`readWrite`
-
-   * - :guilabel:`Schema`
-     - :doc:`Query/View <schema>`
-     - :authaction:`find`
-     - :authrole:`read`
-
-   * - :guilabel:`Documents`
-     - :doc:`Query/View </documents>`
-     - :authaction:`find`
-     - :authrole:`read`
-
-   * - :guilabel:`Documents`
-     - :doc:`Clone, insert, update, delete </documents>`
-     - | :authaction:`find`
-       | :authaction:`insert`
-       | :authaction:`update`
-       | :authaction:`remove`
-     - :authrole:`readWrite`
-
-   * - :guilabel:`Indexes`
-     - :doc:`View </indexes>`
-     - :authaction:`listIndexes`
-     - :authrole:`read`
-
-   * - :guilabel:`Indexes`
-     - :doc:`Create, drop </indexes>`
-     - | :authaction:`createIndex`
-       | :authaction:`dropIndex`
-     - :authrole:`readWrite`
-
-   * - :guilabel:`Explain Plan`
-     - :doc:`Query/View </query-plan>`
-     - :authaction:`find`
-     - :authrole:`read`
-
-   * - :guilabel:`Validation`
-     - :doc:`View rules </validation>`
-     - :authaction:`listCollections`
-     - :authrole:`read`
-
-   * - :guilabel:`Validation`
-     - :ref:`Update rules <update-validation>`
-     - :authaction:`collMod`
-     - :authrole:`dbAdmin`
-
 .. _disconnect:
 
 Disconnect
```

Comment:
I just realized that our formatting is inconsistent across the "Connect" and "Disconnect" sections. In "Connect" we use numbers to format steps, and in "Disconnect" we use bullet point.

Can we:

* Change the bullet points in the "Disconnect" section to numbered steps, as in the "Connect" section?

* Remove ", then" from the first bullet point.

* Perhaps not needed when we change to a number formatting, but can we add a space between the second step and the beginning of the screenshot? It's really tight in that area.

* Finally, I totally forgot to include this the first time, but this screenshot should also match the following acceptance criteria:

> Screenshot is replaced with an up-to-date screenshot of the Compass connection dialog. The new screenshot should have "Recent Connections" cleared (should not show in the screenshot) and should not have "Beta" in the title.

Let me know if that doesn't make sense.

Reply:
Looks great.